scss/themes/_offline.scss

```diff
@@ -20,7 +20,8 @@ body.theme-scarlet.offline,
 body.theme-seafoam.offline,
 body.theme-gecko.offline,
 body.theme-ozark.offline,
-body.theme-cobalt.offline
+body.theme-cobalt.offline,
```

Comment:
Oh, nice. Seems like you also fixed a missing comma here. 👍

Reply:
oops yes, that's a bug